actions/actions.py

```diff
@@ -6,10 +6,6 @@
 
 sys.path.append("lib")
 
-from charms.layer.basic import activate_venv  # NOQA E402
-
-activate_venv()
```

Comment:
I know I've seen this before in charms...  The only worry I have is if there is a reason this existed other than perhaps use_venv not being supported by charm builds back when this charm was first authored.  ...I think this should be OK though.  It looks fine.

Is this a charm tested via testflinger or which is otherwise being run with more than just unit tests?  If so, maybe this is OK.  As long as the charm still runs when built with this modification, I think it's OK.

Reply:
Glad I asked.  I think the sshebang change addresses my concern here.